preview/src/routes/index.svelte

```diff
@@ -69,8 +74,19 @@
               themes: ["white", "g10", "g80", "g90", "g100"],
             }}
           />
+          <Select
+            id="select-preview-size"
+            labelText="Preview size"
```

Comment:
One last nit: I'd like the label to read "Icon size"

Reply:
Ok. Updated all variables and labels in https://github.com/carbon-design-system/carbon-icons-svelte/pull/145/commits/d84e781b2b3a8a1ba975574b24c09329c8c3ee8a as well, to be more straightforward and consistent.